cts-java/src/test/java/org/ga4gh/cts/api/g2p/GenotypePhenotypeSearchIT.java

```diff
@@ -0,0 +1,264 @@
+            // no-op, expected
+        }
+    }
+
+    /**
+     * search for feature
+     * @throws AvroRemoteException
+     */
+    @Test
+    public void simpleFeatureSearch() throws AvroRemoteException {
+        SearchGenotypePhenotypeRequest request = SearchGenotypePhenotypeRequest.newBuilder().build();
+        request.setFeature("KIT *wild");
+        SearchGenotypePhenotypeResponse response = client.genotypePhenotype.searchGenotypePhenotype(request);
+        assertThat(response).isNotNull();
+        assertThat(response.getAssociations()).isNotNull() ;
+        assertThat(response.getAssociations().get(0)).isNotNull() ;
+    }
+
+
+    /**
+     * search for evidence
+     * @throws AvroRemoteException
+     */
+    @Test
+    public void simpleEvidenceSearch() throws AvroRemoteException {
+        SearchGenotypePhenotypeRequest request = SearchGenotypePhenotypeRequest.newBuilder().build();
+        request.setEvidence("imatinib");
+        SearchGenotypePhenotypeResponse response = client.genotypePhenotype.searchGenotypePhenotype(request);
+        assertThat(response).isNotNull();
+        assertThat(response.getAssociations()).isNotNull() ;
+        assertThat(response.getAssociations().get(0)).isNotNull() ;
+    }
+
+
+
+    /**
+     * search for Phenotype
+     * @throws AvroRemoteException
+     */
+    @Test
+    public void simplePhenotypeSearch() throws AvroRemoteException {
+        SearchGenotypePhenotypeRequest request = SearchGenotypePhenotypeRequest.newBuilder().build();
+        request.setPhenotype("GIST");
+        SearchGenotypePhenotypeResponse response = client.genotypePhenotype.searchGenotypePhenotype(request);
+        assertThat(response).isNotNull();
+        assertThat(response.getAssociations()).isNotNull() ;
+        assertThat(response.getAssociations().get(0)).isNotNull() ;
+    }
+
+
+    /**
+     * search for Phenotype
+     * @throws AvroRemoteException
+     */
+    @Test
+    public void simplePhenotypeSearchNoFind() throws AvroRemoteException {
+        SearchGenotypePhenotypeRequest request = SearchGenotypePhenotypeRequest.newBuilder().build();
+        request.setPhenotype("FooBar");
+        SearchGenotypePhenotypeResponse response = client.genotypePhenotype.searchGenotypePhenotype(request);
+        assertThat(response).isNotNull();
+        assertThat(response.getAssociations()).isNotNull() ;
+        assertThat(response.getAssociations().size()).isEqualTo(0) ;
+    }
+
+
+    /**
+     * search for Feature using external id
+     * @throws AvroRemoteException
+     */
+    @Test
+    public void simpleFeatureSearchExternalIdentifier() throws AvroRemoteException {
+        SearchGenotypePhenotypeRequest request = SearchGenotypePhenotypeRequest.newBuilder().build();
+
+        ExternalIdentifier id = new ExternalIdentifier() ;
+        id.setDatabase("http://www.monarchinitiative.org/_");
+        id.setIdentifier("CGD:d8c2d551UniProtKB:P10721#P10721-1Region");
+        id.setVersion("*");
```

Comment:
For these and all subsequent constants that refer to things in your compliance data, please add them to class `TestData`, give them clear, meaningful names, and refer to them in tests by those names.

If you have a constant that's meant _not_ to match something in the compliance data, as when you're trying to provoke an exception, please use a method like `Utils.randomName` or `Utils.randomId`, depending on what's meant, instead of a constant that might accidentally look valid to the casual observer.  It'll make your intention more clear.